EIPS/eip-2566.md

```diff
@@ -0,0 +1,128 @@
+---
+eip: 2566
+title: Human Readable Parameters for Contract Function Execution
+author: Joseph Stockermans (@jstoxrocky)
+discussions-to: https://ethereum-magicians.org/t/human-readable-parameters-for-contract-function-execution/4154, https://github.com/ethereum/EIPs/issues/2567
+status: Draft
+type: Standards Track
+category: Interface
+created: 2020-03-23
+---
+
+## Simple Summary
+This EIP proposes a new Ethereum RPC method `eth_sendTransactionToContract` that accepts a human readable version of `eth_sendTransaction`'s `data` field. This change will allow ProviderWallets (hybrid Ethereum provider / wallet software) like Metamask and Geth to display confirmation screens with human readable details of a function call to a contract.
```

Comment:
Simple summary should be of similar length/content to an email subject line or a forum title or a GitHub PR title.  Currently, this is too long for a Simple Summary and should be shortened up a lot.

Also, the latter portion of this summary is more appropriate for motivation than summary.
```suggestion
 This EIP proposes a new Ethereum RPC method `eth_sendTransactionToContract` that accepts a human readable version of `eth_sendTransaction`'s `data` field.
```